code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/assassination.dm

```diff
@@ -147,19 +167,20 @@
 	id = /obj/item/card/id/job/clown
 	ears = /obj/item/radio/headset/headset_srv
 	uniform = /obj/item/clothing/under/rank/civilian/clown
-	shoes = /obj/item/clothing/shoes/clown_shoes
+	shoes = /obj/item/clothing/shoes/clown_shoes/taeclowndo
 	mask = /obj/item/clothing/mask/gas/clown_hat
 	l_pocket = /obj/item/bikehorn
 	back = /obj/item/storage/backpack/clown
 	backpack_contents = list(
 		/obj/item/stamp/clown = 1,
-		/obj/item/reagent_containers/spray/waterflower = 1,
+		/obj/item/reagent_containers/spray/waterflower/lube = 1,
 		/obj/item/food/grown/banana = 1,
 		/obj/item/instrument/bikehorn = 1,
+		/obj/item/storage/firstaid/infiltrator,
+		/obj/item/flashlight
 		)
 
 	implants = list(/obj/item/implant/sad_trombone)
 
 /datum/outfit/vip_target/clown/post_equip(mob/living/carbon/human/H)
 	H.fully_replace_character_name(H.real_name, pick(GLOB.clown_names))
-	H.dna.add_mutation(CLOWNMUT)
```

Comment:
why are you removing the clown mutation?

Reply:
To quote the PR body:

> He is also no longer clumsy though, so if he can successfully disarm and swipe a weapon from the explorer, that becomes the clown's weapon instead. Still the weakest of the lot, but at least able to do something other than die miserably now.